gremlin-dotnet/src/Gremlin.Net/Driver/GremlinClient.cs

```diff
@@ -108,7 +108,11 @@ public class GremlinClient : IGremlinClient
 
             var connectionFactory =
                 new ConnectionFactory(gremlinServer, messageSerializer,
-                    new WebSocketSettings { WebSocketConfigurationCallback = webSocketConfiguration }, sessionId);
+                    new WebSocketSettings
+                    {
+                        WebSocketConfigurationCallback = webSocketConfiguration,
+                        EnableUserAgentOnConnect = connectionPoolSettings?.EnableUserAgentOnConnect ?? true
```

Comment:
there are `DefaultEnableUserAgentOnConnect` const, better to use it not just `true`